config/routes.rb

```diff
@@ -72,8 +72,8 @@
     put :make_profile_photo
   end
 
-	#Search
-	get 'search' => "search#search"
+  #Search
+  get 'search' => "search#search"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.